lms/static/js/certificates/views/certificate_invalidation_view.js

```diff
@@ -41,10 +42,9 @@
                     });
 
                     if (this.collection.findWhere({user: user})) {
-                        this.showMessage(
-                            gettext("Certificate of ") + user +
-                            gettext(" has already been invalidated. Please check your spelling and retry."
-                            ));
+                        message = gettext("Certificate of <%= user %> has already been invalidated. " +
+                            "Please check your spelling and retry.");
+                        this.showMessage(_.template(message, {user: user}));
```

Comment:
@saleem-latif @nedbat @mattdrayer @andy-armstrong 
Unfortunately, we don't have the proper helper method available yet.  It might be coming in the following story, if you're interested in a sneak preview:
https://openedx.atlassian.net/browse/TNL-3966

That story was about Underscore templates, but it might be the same helper for Coffeescript.

For now, since you don't have any html in this code, you could get away with the following:
```
this.showMessage(_.escape(_.template(message, {user: user})));
```
It would be far better to have the helper, but we don't, so this is what I would propose if others agree.

Also, `showMessage()` seems to call `gettext()` again on the message.  That doesn't seem right to me.